app/controllers/compute_resources_controller.rb

```diff
@@ -36,6 +36,16 @@ def create
       @compute_resource.valid?
       process_error
     end
+  rescue OVIRT::OvirtException => e
```

Comment:
should we add some sort of TODO here? as long term I dont expect to see rbovirt errors in our codebase? (or in other words, I would expect that after @orrabin finish the fog-ovirt rewrite, we will not know rbovirt in foreman code base at all).

Reply:
@ohadlevy this isn't the only place we reference `OVIRT` module or exceptions. Ideally, we shouldn't care about the underlying implementation here and for other resource providers and push anything that does care into fog (I think we also refer to some RbVmomi stuff elsewhere).
Might be a good idea to open a separate issue to remove all references once `fog-ovirt` handles them. What do you think @orrabin ?

Reply:
That is the plan. Once we start using the version of `fog-ovirt` that wraps exceptions we will change all references to `OVIRT::OvirtException` in foreman. There isn't an issue for it yet, I'll open it.